src/quo2/components/token_tag.cljs

```diff
@@ -0,0 +1,68 @@
+(ns quo2.components.token-tag
+  (:require [quo2.foundations.colors :as colors]
+            [quo.react-native :as rn]
+            [quo.theme :as theme]
+            [status-im.ui.components.icons.icons :as icons]
+            [quo2.components.text :as text]))
+
+(def themes {:light   {:background-color colors/neutral-20}
+             :dark  {:background-color colors/neutral-80}})
+
+(defn get-value-from-size [size, big-option, small-option]
+  (if  (= size :big) big-option small-option))
+
+(defn style-container [size border-color is-required]
+  (merge {:height             32
+          :min-width         (get-value-from-size size 94 76)
+          :align-items        :center
+          :flex-direction     :row
+          :left             0
+          :border-radius      100
+          :padding-right      10}
+         (if is-required {:border-color     border-color
+                          :border-width     1} {})))
```

Comment:
redundant else clause, can be replaced with when

Reply:
thanks, adjusted